tests/test_testbox_workflow_security.py

```
@@ -0,0 +1,285 @@
+
+class TestboxBrokerSecurityTests(unittest.TestCase):
+    def workflow_text(self) -> str:
+        self.assertTrue(
+            WORKFLOW_PATH.is_file(),
+            "main-controlled Testbox broker workflow is missing: "
+            f"{WORKFLOW_PATH.relative_to(ROOT)}",
+        )
+        return WORKFLOW_PATH.read_text(encoding="utf-8")
+
+    def workflow_document(self) -> dict[str, Any]:
+        document = yaml.load(self.workflow_text(), Loader=yaml.BaseLoader)
```

Ruff (0.16.1)

[error] Probable use of unsafe loader `BaseLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)